app/controllers/provider/request_password_resets_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Provider::RequestPasswordResetsController < FrontendController
```

Comment:
There is already a password controller, why can't we use it?

https://github.com/3scale/porta/blob/8b27ca70d7700d9c275401439e44ea9ff6aa639e/app/controllers/provider/passwords_controller.rb

Reply:
this passwords controller already has this custom logic for the `new` action that regenerates the token.

Reply:
I know but can we reuse the same controller?

Reply:
@hallelujah I moved it to the already existent controller